### PR TITLE
seding multiple invitation asynchronously

### PR DESCRIPTION
**Pull request by yogeshjain999**
Emails and roles can be specified in text fields or in uploaded .csv file

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,12 +15,17 @@ gem 'jquery-rails'
 gem 'sass-rails', '>= 3.2'
 gem 'bootstrap-sass'
 gem 'mongoid', github: 'mongoid/mongoid'
-gem 'mongoid_slug'
 gem 'haml-rails'
 gem 'cancan'
 gem 'coffee-rails'
 gem 'therubyracer'
 gem 'bootstrap-datepicker-rails'
+gem 'sinatra', require: false
```
Comment (kirandc) [inline Gemfile:23]: Why sinatra require?

Comment (yogeshjain999) [inline Gemfile:23]: For sidekiq UI

File: office_automation_employee.gemspec [hunk 2]
```diff
@@ -7,7 +7,7 @@ require "office_automation_employee/version"
 Gem::Specification.new do |s|
   s.name        = "office_automation_employee"
   s.version     = OfficeAutomationEmployee::VERSION
-  s.authors     = ["yogesh", "pranav", "shweta"]
+  s.authors     = ["yogesh","pranav", "shweta"]
```
Comment (kirandc) [inline office_automation_employee.gemspec]: Remove pranav name

File: app/views/office_automation_employee/devise/invitations/new.html.haml [hunk 3]
```diff
@@ -9,17 +9,28 @@
...
+            #invitee
+              = f.fields_for :users do |user|
+                - if user.object.new_record?
+                  .form-group
+                    .row
+                      .col-md-5
+                        = user.input :email, autofocus: true, input_html: { class: "form-control", placeholder: "Enter Email address", maxlength: 50}, label: false
+                      .col-md-3
+                        = user.input :roles, collection: OfficeAutomationEmployee::Role.all, input_html:{ class: "form-control" }, label: false, selected: true
+                      .col-md-4
```
Comment (kirandc) [inline app/views/office_automation_employee/devise/invitations/new.html.haml:29]: Provide csv format help on invitation page

File: app/assets/javascripts/office_automation_employee/invitations.js [hunk 4]
```diff
@@ -1,2 +1,11 @@
-// Place all the behaviors and hooks related to the matching controller here.
-// All this logic will automatically be available in application.js.
+$(function() {
```
Comment (kirandc) [inline app/assets/javascripts/office_automation_employee/invitations.js:1]: Those JS file you writing always write in coffeescript. replace this file.

**Pull request merged**